packages/@react-types/menu/src/index.d.ts

```diff
@@ -17,22 +17,39 @@ export type FocusStrategy = 'first' | 'last';
 
 export interface MenuTriggerProps {
   // trigger?: 'press' | 'longPress',
+  /** Where the Menu aligns with it's trigger. */
   align?: Alignment,
-  direction?: 'bottom' | 'top', // left right?
+  /** Where the Menu opens relative to it's trigger. */
+  direction?: 'bottom' | 'top',
+  /** Whether the Menu closes when a selection is made. */
   closeOnSelect?: boolean,
+  /** Whether the Menu loads open (controlled). */
   isOpen?: boolean,
+  /** Whether the Menu loads open (uncontrolled). */
   defaultOpen?: boolean,
+  /** Handler that is called when the Menu opens or closes. */
   onOpenChange?: (isOpen: boolean) => void,
+  /**
+   * Whether the element should flip its orientation when there is insufficient
+   * room for it to render completely.
```

Comment:
Instead of saying 'render completely' IMO I like the way you described it in L391 & 392 of MenuTrigger.mdx where you said "... where the original placement would cause it to render out of view."